sentry/src/routes/routers.rs

```diff
@@ -133,12 +135,28 @@ async fn if_dummy_adapter<C: Locked + 'static, B>(
 }
 
 pub fn channels_router_axum<C: Locked + 'static>() -> Router {
+    let spender_routes = Router::new()
+        .route(
+            "/:addr",
+            get(get_spender_limits_axum::<C>)
+                .post(add_spender_leaf_axum::<C>)
+                .route_layer(middleware::from_fn(authentication_required::<C, _>)),
+        )
+        .route(
+            "/all",
+            get(get_all_spender_limits_axum::<C>)
+                .route_layer(middleware::from_fn(authentication_required::<C, _>)),
+        );
```

Comment:
since both routes require authentication, you can add the `layer()` om the Router itself, instead of on each route